src/plugins/data/common/field_formats/converters/numeral.ts

```diff
@@ -51,10 +57,24 @@ export abstract class NumeralFormat extends FieldFormat {
     if (isNaN(val)) return '';
 
     const previousLocale = numeral.language();
-    const defaultLocale = (this.getConfig && this.getConfig('format:number:defaultLocale')) || 'en';
-    numeral.language(defaultLocale);
+    const defaultLocale = this.getConfig && this.getConfig('format:number:defaultLocale');
```

Comment:
Using optional chaining syntax:

```suggestion
    const defaultLocale = this.getConfig?.('format:number:defaultLocale');
```

Reply:
This doesn't validate, are you sure that's the right syntax?

Reply:
That should validate. Is your editor maybe not using the workspace TypeScript version (3.7+), but instead a built in pre 3.7 version?